sections/sentinels.tex

```diff
@@ -0,0 +1,183 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Cooperative scenario}{Sentinels}{\images/sentinel.png}
```

Comment:
Wow, this header looks fantastic, nice work 👍🏻

Reply:
BTW, where is the graphic coming from? Is it safe to use?

Reply:
A little bit of dall-e, a little bit of photoshop :) I was not sure whether to put the high-res image or not.

Original (hi-res)             |  Slightly lower quality
:-------------------------:|:-------------------------:
![sentinel](https://github.com/qwrtln/Homm3BG-mission-book/assets/1928291/9b8f762e-fd68-48b8-b186-01ea80b3ef26)  |   ![sentinel](https://github.com/qwrtln/Homm3BG-mission-book/assets/1928291/d2ba6a06-dd43-4171-9eb5-e62b3f885d44)

I wanted to have this oil-paint effect or whatever it is to get the Heroes look & feel. Not sure how much successful I was though

Reply:
Looks very good. I don't mind hi-res. Ghostscript does a very good job with overall PDF size compression (it shrunk the rule book from 77 MB to 22 without any loss of quality). As long as we're not approaching GitHub's 100 MB file size limit, I say go for the hi-res.

Reply:
This one has 1.5MB but it was more about getting the HOMM feel. I put the full quality back until I find a way to create similar graphics